docs/querying/api.md

```diff
@@ -381,3 +381,70 @@ $ curl http://localhost:9090/api/v1/alertmanagers
   }
 }
 ```
+
+
+## TSDB Admin APIs
+These are APIs that expose database functionalities for the advanced user. These APIs are not enabled unless the `--web.enable-admin-api` is set.
+
+We also expose a gRPC API whose definition can be found [here](https://github.com/prometheus/prometheus/blob/master/prompb/rpc.proto). This is experimental and might change in the future.
+
+### Snapshot
+Snapshot creates a snapshot of all current data into `snapshots/<datetime>-<rand>` under the TSDB's data directory and returns the directory as response.
+
+```
+POST /api/v1/admin/tsdb/snapshot
+```
+
+```json
+$ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/snapshot
+{
+  "status":"success",
+  "data":"2017-12-07T07:48:41Z-5bc257d3bf7e7dd2"
```

Comment:
I'd put a `name` field inside the `data`. We may wish to add additional return values at some point, such as duration or total size.

Reply:
Done.